WordPressVIPMinimum/Tests/Variables/VariableAnalysisUnitTest.php

```diff
@@ -31,9 +31,16 @@ public function getErrorList() {
 	 * @return array <int line number> => <int number of warnings>
 	 */
 	public function getWarningList() {
-		return array(
+		$warningList = [
 			5 => 2,
-		);
+		];
+	
+ 		// PHP prior to version 7.x does not properly process the $e.
```

Comment:
There's a leading space at the start of this line.

Reply:
Is there any reference that could be added, to find out more about why it doesn't properly process the `$e`?

Could the code that does the processing be updated, so it does?

Reply:
With VIP running a later version, would it even be applicable to check that PHP is < 7?

Reply:
This repo supports [down to PHP 5.6](https://github.com/Automattic/VIP-Coding-Standards/blob/master/composer.json#L18) (related: https://github.com/Automattic/VIP-Coding-Standards/issues/211).

Until the level is raised to 7.0+ (or later), then yes, it should be checked.

The version that these sniffs are run at, and the version that the code is expected to run at, can be two different versions.

Reply:
> Is there any reference that could be added, to find out more about why it doesn't properly process the $e?

I couldn't find any reference on _why_ and speaking to @david-binda, he doesn't recall why either.

Reply:
Update: I don't believe this is an issue anymore since it appears that the version check was added because it was failing the tests.